src/tools/deploy.ts

```diff
@@ -11,7 +11,11 @@ export default async function deploy(
   // Setup log level
   log.level = process.env.AUTH0_DEBUG === 'true' ? 'debug' : 'info';
 
-  log.info('Getting access token for ' + config('AUTH0_CLIENT_ID') + '/' + config('AUTH0_DOMAIN'));
+  log.info(
+    `Getting access token for ${
+      config('AUTH0_CLIENT_ID') !== undefined ? `${config('AUTH0_CLIENT_ID')}/` : ''
```

Comment:
While I was at it, I fixed the logging statement here to avoid it printing out "undefined" if `AUTH0_CLIENT_ID` is not explicitly defined.